process/accounts/accountsProcessor.go

```diff
@@ -265,6 +260,28 @@ func (ap *accountsProcessor) getESDTInfo(accountESDT *data.AccountESDT) (*big.In
 	return esdtToken.Value, hex.EncodeToString(esdtToken.Properties), tokenMetaData, nil
 }
 
+// PutTokenMedataDataInTokens will put the TokenMedata in provided tokens data
+func (ap *accountsProcessor) PutTokenMedataDataInTokens(tokensData []*data.TokenInfo) {
+	for _, tokenData := range tokensData {
+		if tokenData.Data != nil || tokenData.Nonce == 0 {
+			log.Warn("token data is not nil")
```

Comment:
please log also the identifier, nonce

Reply:
also, it might worth not continuing, but overwriting the metadata?

Reply:
Removed this log because is no longer needed.